code/modules/medical/diseases/lungrot.dm

```diff
@@ -0,0 +1,79 @@
+/datum/ailment/disease/lungrot
+	name = "Lungrot"
+	max_stages = 5
+	stage_prob = 4
+	spread = "Non-Contagious"
+	cure = "Robustissin application after ablation of salbutamol."
```

Comment:
Not sure `ablation` is the right word here? It's a bit confusing anyway, maybe just put `removal` or something.

Reply:
Ablation in english generally refers to gradual destruction by chipping away at the edges of thing, which is sort of correct? But I think it's meant for solid objects and not chemicals.
The word is notably used in heat shields for spacecraft, they're usually made of ablative material and disintegrate to get rid of heat more effectively.